service-manual/the-team/learning-and-development/open-programme.md

```diff
@@ -134,5 +134,5 @@ We need to know that users of digital services are who they say they are. In thi
 
 #Request a place
 
-The GDS Skills Team will invite all service manager induction and foundation day participants to book their places on the open programme. If you have not completed either of these, but are very familiar with digital in government and wish to book straight onto the open programme, please contact the Skills Team for an invitation: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:service-manager-programme@digital.cabinet-office.gov.uk)
+The GDS Skills Team will invite all service manager induction and foundation day participants to book their places on the open programme. If you have not completed either of these, but are very familiar with digital in government and wish to book straight onto the open programme, please contact the Skills Team for an invitation: [digital.academy@dwp.gsi.gov.uk](mailto:digital.academy@DWP.GSI.GOV.UK)
```

Comment:
Whilst this will work with the capitalised portion of the address, it doesn't necessarily need to be, and I think it should be downcased. This doesn't block this getting merged, but if you'd be up for fixing it (I can help, if you like), then that would be good :)